consensus/istanbul/announce/util.go

```diff
@@ -0,0 +1,22 @@
+package announce
+
+import (
+	"fmt"
+
+	"github.com/celo-org/celo-blockchain/consensus/istanbul"
+	"github.com/celo-org/celo-blockchain/p2p/enode"
+)
+
+// DecryptEnodeURL decrypts an encrypted enodeURL with the given ecdsa key pair.
```

Comment:
I think naming to `ProcessEnodeURL` might be more appropriate according to error message at `consensus/istanbul/announce/manager.go:L208`

Reply:
makes sense

Reply:
I actually changed it to DecryptAndParse, I think it makes more sense